src/app/admin/borrowers/borrowers.component.html

```diff
@@ -40,4 +37,66 @@ <h3>Borrowers</h3>
       </tbody>
     </table>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<!-- create borrower modal -->
+<ng-template #createBorrowerModal let-c="close" let-d="dismiss">
```

Comment:
Does this actually work? I found I had to make the modal dialogs separate components to get them to work properly.

Reply:
Yes, it does work. Really, do you mean closing the modal?

Reply:
I think I remember having to make some adjust meant in order for it to work. But I cannot remember exactly what that was.

Reply:
If I remember correctly (which I'm not certain I do), when I tried a `<ng-template>` in another component, I couldn't get it to *open* at all; if it worked even *that* much, I couldn't get it to close or get the "submit" button to do anything. But if it works for you, then :+1:.

Reply:
It would not open period? When you called the open function, did you pass it the id of the <ng-template>? Yes, for the close I needed to make a function that the buttons called, which simply, does `this.modalReference.close();` where the modalReference is an instance of `NgbModal`. I believe I also had to change the submit button as well.